app/models/conference.rb

```diff
@@ -858,7 +858,7 @@ def cfp_set?
   # * +True+ -> If conference has a start and a end date.
   # * +False+ -> If conference has no start or end date.
   def registration_date_set?
-    !!registration_period && !!registration_period.start_date && !!registration_period.end_date
+    registration_period.present? && registration_period.start_date.present? && registration_period.end_date.present?
```

Comment:
`present?` here is not needed. The `registration_period` is an object and `start_date` and `end_date` are times, so `present? ` can only be `false` if those things are `nil`

So:
``` Ruby
registration_period && registration_period.start_date && registration_period.end_date
```